src/exercises/chromium/third-party.md

```diff
@@ -0,0 +1,24 @@
+# Exercise
+
+Add [uwuify][0] to Chromium, turning off the crate's [default features][1].
+Assume that the crate will be used in shipping Chromium, but won't be used
+to handle untrustworthy input.
+
+(In the next exercise we'll use uwuify from Chromium, but feel free to
+skip ahead and do that now if you like. Or, you could create a new
+[`rust_executable` target][2] which uses `uwuify`).
+
+<details>
+Students will need to download lots of transitive dependencies. This will
+appear repetitive - that is indeed correct. There's no shortcut but to
+run lots of separate download commands.
```

Comment:
I haven't yet tried going through the exercises myself, but is that still true?  Are there still multiple "download" commands to run?  Or do we now just need a single "vendor" comment?

Reply:
Yep this was out of date, thanks, fixed.